python/tests/test_topology.py

```diff
@@ -7917,3 +7917,69 @@ def test_is_isolated_bad(self):
                 tree.is_isolated("abc")
             with pytest.raises(TypeError):
                 tree.is_isolated(1.1)
+
+
+class TestExampleTrees:
+    """
+    Test the creation of hard-coded example tree (sequences)
+    """
+
+    def test_star_equivalent(self):
+        for extra_params in [{}]:  # , {'span':2.5}]: # Add after #930 is merged
```

Comment:
There's only one extra param, so loop seems unnecessary and makes the code less readable

Reply:
I've just uncommented the bit with "span" in, so it will test with with the default, and with a span now.